client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/form-content/sms-notification.tsx

```diff
@@ -46,9 +61,12 @@ export default function SMSNotification( {
 							{ translate( 'You need at least one phone number' ) }
 						</AlertBanner>
 					</div>
-					<ConfigureSMSNotification toggleModal={ toggleModal } />
+					<ConfigureSMSNotification toggleModal={ toggleModal } allPhoneItems={ allPhoneNumbers } />
 				</>
 			) }
+			{ enableSMSNotification && allPhoneNumbers.length > 0 && (
+				<ConfigureSMSNotification toggleModal={ toggleModal } allPhoneItems={ allPhoneNumbers } />
+			) }
```

Comment:
It looks like the conflicts are not resolved correctly.

These are not needed to be added again since it is already being rendered on top.